mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+			  <single></single>
```

Comment:
@liammr, by the way, are you missing any other locator terms besides "article"? Frank Bennett already added "article", "rule", and "title" to his fork of CSL for improved legal support, and I was wondering if you need any others.